.clang-format

```diff
@@ -0,0 +1,52 @@
+Language: Cpp
+Standard: c++20
+
+BasedOnStyle: Chromium
+
+ColumnLimit: 120
```

Comment:
Personally, I would prefer a tighter limit of 100 to comfortably fit three code windows on my screen. Both for editing and also side-by-side reviews on GH.

A quick glance through the code didn't seem to make things too crammed with '100' either.

Reply:
That's fine. With either 100 or 120 limit, some things don't come out as I'd like. Either way the fix is to over time rewrite those bits of code so that it formats nicely.